#! /bin/sh /usr/share/dpatch/dpatch-run
# 24_bscan-wrong-and-missing-opts.dpatch
#
# Subject: Fix -p/-P in bscan.8 man page
# Origin: upstream, http://bacula.git.sourceforge.net/git/gitweb.cgi?p=bacula/bacula;a=commit;h=30cb927b27f568fe643a7b627c0edaf844aad301
# Bug: http://bugs.bacula.org/view.php?id=1373
# Bug-Debian: http://bugs.debian.org/cgi-bin/bugreport.cgi?bug=499619
# Forwarded: http://bugs.bacula.org/view.php?id=1373
# Author: Carsten Leonhardt <leo@carotec.de>
# Reviewed-by: Eric Bollengier <eric@eb.homelinux.org>
# Last-Update: 2009.12.3
@DPATCH@
diff --git a/bacula/manpages/bscan.8 b/bacula/manpages/bscan.8
index 3d9d6c3..ee7e997 100644
--- a/bacula/manpages/bscan.8
+++ b/bacula/manpages/bscan.8
@@ -58,7 +58,7 @@ Specify the database name (default: \fIbacula\fP)
 .BI \-u\  username
 Specify database username (default: \fIbacula\fP)
 .TP
-.BI \-p\  password
+.BI \-P\  password
 Specify database password (default: \fInone\fP)
 .TP
 .BI \-h\  host
@@ -73,6 +73,9 @@ Synchronize or store in Database.
 .B \-v
 Verbose output mode.
 .TP
+.B \-p
+Proceed inspite of I/O errors
+.TP
 .BI \-V\  volume
 Specify volume names (separated by '|')
 .TP